esmvaltool/diag_scripts/validation.py

```diff
@@ -36,42 +36,90 @@ def plot_contour(cube, plt_title, file_name):
     plt.close()
 
 
-def plot_latlon_cubes(cube_1, cube_2, cfg, data_names, obs_name=None):
+def save_plotted_cubes(cube, cfg, plot_name):
+    """Save cubes that were plotted to disk."""
+    if "save_cubes" in cfg:
+        if cfg["save_cubes"]:
```

Comment:
```suggestion
        if cfg["write_netcdf"]:
```
the option can be set in config-user.yml

Reply:
this is an extra option to be set by the user - they may not need files at all :beer:

Reply:
How is this option different from not saving the netcdf files configured in config-user?

Reply:
you can turn it on and off for each of the sub-diagnostic optionals (scripts)

Reply:
Ok, if it makes sense for you to have such fine grained control, I think this should be fine.